yarn.lock

```diff
@@ -17043,7 +17043,7 @@ vue-lazy-hydration@^1.0.0-beta.9:
   resolved "https://registry.yarnpkg.com/vue-lazy-hydration/-/vue-lazy-hydration-1.0.0-beta.12.tgz#d578dc86059eb0a147e6e553585dd22c41dfbae2"
   integrity sha512-0innXtjaqNodh0aw1/3dgGdEfL76eqDzo3CETSKolyjm3Z++hqBS7GQkrAAfp4JYBE12mb5yP2YnJxK6tJSGIA==
 
-vue-lazyload@^1.2.6, vue-lazyload@^1.3.3:
+vue-lazyload@^1.2.6:
```

Comment:
`vue-observe-visibility` did not disappear. Probably because of not using it from both package.json's